CMakeTests/AddCompileDefinitions.cmake

```diff
@@ -0,0 +1,42 @@
+# Add a C or C++ compile definitions to the current scope
+#
+# add_compile_definitions(def [def ...] [DEBUG | RELEASE])
```

Comment:
Same comment as on `check_and_add_flags`. DEBUG and RELEASE are hard for me to scan, for some reason.